<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<comment> Begin CVS Header </comment>
<comment>   $Source: /Volumes/Home/Users/shoops/cvs/copasi_dev/copasi/UI/CUpDownSubwidget.ui,v $ </comment>
<comment>   $Revision: 1.7 $ </comment>
<comment>   $Name:  $ </comment>
<comment>   $Author: shoops $ </comment>
<comment>   $Date: 2011/03/07 19:38:03 $ </comment>
<comment> End CVS Header </comment>

<comment> Copyright (C) 2011 - 2010 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., University of Heidelberg, and The University </comment>
<comment> of Manchester. </comment>
<comment> All rights reserved. </comment>

 <class>CUpDownSubwidget</class>
 <widget class="QWidget" name="CUpDownSubwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>120</width>
    <height>60</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="focusPolicy">
   <enum>Qt::WheelFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form2</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>3</number>
   </property>
   <item>
    <layout class="QGridLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QToolButton" name="toolButtonDel">
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>image0</normaloff>image0</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="toolButtonUp">
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>image1</normaloff>image1</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QToolButton" name="toolButtonCopy">
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>image2</normaloff>image2</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="toolButtonDown">
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>image3</normaloff>image3</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>6</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>toolButtonDown</sender>
   <signal>clicked()</signal>
   <receiver>CUpDownSubwidget</receiver>
   <slot>slotDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonUp</sender>
   <signal>clicked()</signal>
   <receiver>CUpDownSubwidget</receiver>
   <slot>slotUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonDel</sender>
   <signal>clicked()</signal>
   <receiver>CUpDownSubwidget</receiver>
   <slot>slotDel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonCopy</sender>
   <signal>clicked()</signal>
   <receiver>CUpDownSubwidget</receiver>
   <slot>slotCopy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
